src/app/login/page.tsx

```diff
@@ -1,7 +1,5 @@
-import Image from "next/image";
-
-import FacebookLogo from "./FacebookLogo.svg";
-import GoogleLogo from "./GoogleLogo.svg";
+import FacebookLogo from "/public/icons/FacebookLogo.svg";
+import GoogleLogo from "/public/icons/GoogleLogo.svg";
```

Comment:
czemu w dalszym ciągu importujemy svg zamiast korzystać z generowania ikon za pomocą svgr i tworzenia komponentów reactowych?

Reply:
Sugerowałem się tym fragmentem z dokumentacji SVGR, gdzie po zainstalowaniu paczki i zmodyfikowaniu next.config.js pokazany jest przykład użycia właśnie w ten sposób 
https://react-svgr.com/docs/next/
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/f6632a91-3262-41e2-9a7a-fc8891c449d3)

Reply:
Dodam jeszcze, że fragment kodu do wklejenia w next.config.js podany na stronie nie działał. Sięgnąłem więc po poradnik na YouTube w którym jest on inny:
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/46315b1e-cf74-457c-8d56-212bf8963c18)
https://www.youtube.com/watch?v=mkkfd9KJjRo&t=59s
 i zaczęło działać. Teraz po załadowaniu tej gałęzi dostaję błąd: 
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/354ea392-58c1-4f68-beb5-12a7b6383217)
Więc najwyraźniej coś jednak zrobiłem źle :/

Reply:
Czy chodzi o takie użycie?
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/0d205b12-ef6b-4f5b-bdb4-1f83dacb2fe4)

Reply:
Poza dokumentacją przejrzałem jeszcze kilka blogów i wszędzie wskazuje się na używanie svgr w ten sam sposób 😕
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/bd964052-b0e4-49d6-8c3f-91aed107e3f6)
![obraz](https://github.com/mikolajgraczyk/game-center-store/assets/118211320/27657b11-8ded-4895-ac26-4bce3ad6c055)
Jeżeli u nas ma się to odbywać jakoś inaczej to proszę o nakierowanie 😉

Reply:
ok, to zostawmy w ten sposób, to widocznie zaktualizowali sposób korzystania z libki 👍